docs/administrator-guide/configuration/enterprise-edition/workers/README.md

```diff
@@ -22,19 +22,20 @@ kestra:
 ```
 
 ### `kestra.ee.java-security.forbidden-paths`
-This is the filepath on the system that the Kestra Worker will be forbidden to read or write at all times. This can be useful to protect Kestra configuration files for example.
+This is a list of path on the file system that the Kestra Worker will be forbidden to read or write to. This can be useful to protect Kestra configuration files for example.
 
 ### `kestra.ee.java-security.authorized-class-prefix`
-This security prevents access to file system or thread creation (that would prevent us to check file system access). Most of the plugins will never need to have file system access (except on temporary directories that are created & isolated for a task, and are whitelisted by default). Here you can set a list of prefixes (namespace) classes that will be allowed, all others will be refused.
+This is a list of classes  that can access the file system or create threads.
+Most of the plugins will never need to have access to the file system (except on temporary directories that are created and isolated for each task, and are whitelisted by default). Here you can set a list of prefixes (namespace) classes that will be allowed, all others will be refused.
 
 For example, [GCP plugins](/plugins/plugin-gcp/) will need to create a thread in order to reach the GCP api. Since this whole plugin is deemed safe, you can whitelist it.
 
 ::: warning
-At this time, all the Kestra official plugins are safe to be whitelisted **except** [all scripts plugins](/plugins/plugin-script-groovy/) since it allows custom code to be created that can read & write on the file system. This one must not be added to the `authorized-class-prefix`
+At this time, all the Kestra official plugins are safe to be whitelisted **except** [all scripts plugins](/plugins/plugin-script-groovy/) since they allow custom code to be created that can read and write on the file system. These ones must not be added to the `authorized-class-prefix`
```

Comment:
```suggestion
Currently, all the Kestra official plugins are safe to be whitelisted **except** [all scripts plugins](/plugins/plugin-script-groovy/) since they allow custom code to be created that can be read and written on the file system. These must not be added to the `authorized-class-prefix`
```